README.md

```diff
@@ -1,46 +1,81 @@
 
-**Note**: This extension only supports Microsoft Edge
+**Note**: This extension _only_ supports Microsoft Edge
 
-![Microsoft Edge Tools - Demo](demo.gif)
+![Microsoft Edge Tools - Demo](basic_usage.gif)
+
+## Supported Features
 
-**Supported Features**
 * Debug configurations for launching Microsoft Edge browser in remote-debugging mode and auto attaching the tools,
 * Side Bar view for listing all the debuggable targets, including tabs, extensions, service workers, etc.
 * Fully featured Elements and Network tool with views for HTML, CSS, accessibility and more.
-* Screen-casting feature to allow you to see your page without leaving VS Code.
+* Screen-casting feature to allow you to see your page without leaving Visual Studio Code.
 * Go directly to the line/column for source files in your workspace when clicking on a link or CSS rule inside the Elements tool.
 * Auto attach the Microsoft Edge Tools when you start debugging with the [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) extension.
+* Debug using a windowed or headless version of the Microsoft Edge Browser
+
+## Using the Extension
+
+### Getting Started
 
-# Using the Extension
-## Getting Started
-For use inside VS Code:
+For use inside Visual Studio Code:
 
 1. Install any channel (Canary/Dev/etc.) of [Microsoft Edge](https://aka.ms/edgeinsider).
 1. Install the extension [Microsoft Edge Tools](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools).
 1. Open the folder containing the project you want to work on.
 1. (Optional) Install the [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) extension.
 
-## Using the tools
-The extension operates in two modes - it can launch an instance of Microsoft Edge navigated to your app, or it can attach to a running instance of Microsoft Edge. Both modes requires you to be serving your web application from local web server, which is started from either a VS Code task or from your command-line. Using the `url` parameter you simply tell VS Code which URL to either open or launch in the browser.
+### Changing Extension Settings
+
+You can customize the extension to your needs. You can reach the settings by clicking the gear icon of the extension listing or via the settings menu.
+
+![The gear icon in the extension listing](edge-devtools-for-vscode-settings.png)
+
+#### Turning on Network Inspection
+
+You can enable the Network Pane to inspect any network request of the attached browser. To do this, change the setting and restart the extension.
+
+![Example how to turn on the network pane](edge-devtools-for-vscode-settings-network.png)
+
+You can see an example of the change in the following screencast. After restart, you get an extra tab with network functionality.
+
+![Example how to turn on the network pane](basic_usage(network_and_headless).gif)
+
+#### Turning on Headless Mode
```

Comment:
I like this section being here for visibility but probably makes more sense at the bottom. I don't have a strong opinion on this so I'm fine with either.